app/routes.go

```diff
@@ -16,5 +16,6 @@ func setRoutes(root *buffalo.App) {
 
 	root.GET("/", actions.Index)
 	root.GET("/todo/new", actions.NewTodo)
+	root.POST("/todo/store", actions.StoreTodo)
```

Comment:
This is nice, but our convention is as follows:
root.POST("/todo", actions.SaveTodo)